help_to_heat/frontdoor/routing/forwards_routing.py

```diff
@@ -0,0 +1,500 @@
+from help_to_heat.frontdoor.consts import (
+    address_choice_field,
+    address_choice_field_enter_manually,
+    address_choice_field_write_address,
+    address_manual_page,
+    address_page,
+    address_select_choice_field,
+    address_select_choice_field_select_address,
+    address_select_page,
+    benefits_field,
+    benefits_page,
+    bulb_warning_page,
+    bulb_warning_page_field,
+    confirm_and_submit_page,
+    contact_details_page,
+    council_tax_band_field,
+    council_tax_band_page,
+    country_field,
+    country_field_england,
+    country_field_northern_ireland,
+    country_field_scotland,
+    country_field_wales,
+    country_page,
+    duplicate_uprn_field,
+    epc_accept_suggested_epc_field,
+    epc_found_field,
+    epc_ineligible_page,
+    epc_page,
+    epc_rating_is_eligible_field,
+    epc_select_choice_field,
+    epc_select_choice_field_enter_manually,
+    epc_select_choice_field_epc_api_fail,
+    epc_select_choice_field_select_epc,
+    epc_select_page,
+    field_no,
+    field_yes,
+    household_income_field,
+    household_income_page,
+    loft_access_field,
+    loft_access_page,
+    loft_field,
+    loft_field_no,
+    loft_field_yes,
+    loft_insulation_field,
+    loft_insulation_page,
+    loft_page,
+    northern_ireland_ineligible_page,
+    number_of_bedrooms_field,
+    number_of_bedrooms_page,
+    own_property_field,
+    own_property_field_social_housing,
+    own_property_fields_non_social_housing,
+    own_property_page,
+    park_home_field,
+    park_home_ineligible_page,
+    park_home_main_residence_field,
+    park_home_main_residence_page,
+    park_home_page,
+    property_ineligible_page,
+    property_subtype_field,
+    property_subtype_page,
+    property_type_field,
+    property_type_page,
+    referral_already_submitted_page,
+    schemes_page,
+    success_page,
+    summary_page,
+    supplier_field,
+    supplier_field_british_gas,
+    supplier_field_bulb,
+    supplier_field_e,
+    supplier_field_edf,
+    supplier_field_eon_next,
+    supplier_field_foxglove,
+    supplier_field_octopus,
+    supplier_field_ovo,
+    supplier_field_scottish_power,
+    supplier_field_shell,
+    supplier_field_utilita,
+    supplier_field_utility_warehouse,
+    supplier_page,
+    unknown_page,
+    utility_warehouse_warning_page,
+    utility_warehouse_warning_page_field,
+    wall_insulation_field,
+    wall_insulation_page,
+    wall_type_field,
+    wall_type_page,
+)
+from help_to_heat.frontdoor.eligibility import calculate_eligibility
+
+_unknown_response = unknown_page, False
+
+
+def _requires_answer(answer_key):
```

Comment:
in many cases the routing will enforce this since it requires the answer to know which page to send to next. it's not needed in all cases so I could remove it where it's not needed

Reply:
the pattern it replaces was something along the lines of
```py
def get_some_page():
    if answers.get(some_answer) is None:
        return unknown_page
    # rest of page logic
```

Reply:
Sure, let's leave this